tests/classes_test.py

```diff
@@ -1399,7 +1399,11 @@ def test_name(self):
         self.assertEqual(self.node.name, 'Hello')
 
     def test_userData(self):
-        self.assertEqual(1, self.node.userData["rememberToMakeCoffee"])
+        if int(self.font.appVersion) >= 1075:
```

Comment:
The unit tests should not contain any logic like this. The input file is supposed to be fixed, and the test should only have code that test the current input file (GlyphsUnitTestSans). If we wanted to support several versions of Glyphs, I think we should have several version of GlyphsUnitTestSans and several tests files, one per version, that test specific aspects of that version. But anyway, I don't think we want to keep the API compatible with several versions of Glyphs.